owlbot.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Google LLC
```

Comment:
Make this a `git mv` or change to current year?

Reply:
I did `git mv` but git doesn't preserve it:
https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history

I'd like to keep the existing copyright because at least one line (the doc comment) has been preserved from 2018.